File: app/src/main/java/com/tutpro/baresip/HistoryRow.java
package com.tutpro.baresip;

public class HistoryRow {

    private String peer_uri;
    private Integer direction;
    private String time;

    public HistoryRow(String peer_uri, Integer direction, String time) {
        this.peer_uri = peer_uri;
        this.direction = direction;
        this.time = time;
    }

    public String getPeerURI() { return peer_uri; }
    public Integer getDirection() { return direction; }
    public String getTime() { return time; }

}

File: app/src/main/kotlin/com/tutpro/baresip/HistoryRow.kt
package com.tutpro.baresip

class HistoryRow(val peerURI: String, val peerDomain: String, val direction: Int, val time: String,
                 val index: Int) {

    val directions = ArrayList<Int>()
    val indexes = ArrayList<Int>()

    init {
        directions.add(direction)
        indexes.add(index)
    }
}
